client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/step-content.tsx

```diff
@@ -9,11 +12,79 @@ type StepContentProps = {
 	goToStep?: NavigationControls[ 'goToStep' ];
 };
 
-const StepContent = ( { siteSlug, submit, goNext, goToStep }: StepContentProps ) => (
-	<div className="launchpad__content">
-		<Sidebar siteSlug={ siteSlug } submit={ submit } goNext={ goNext } goToStep={ goToStep } />
-		<LaunchpadSitePreview siteSlug={ siteSlug } />
-	</div>
-);
+enum UseFor {
```

Comment:
I created this enum for the sake of only having a single function `findBestDomain` as opposed to having two `findBestDomainForIFrame` and `findBestDomainForPreview` because they would pretty much duplicate all of the code with the exception of exiting early when we find the `.wordpress.com` domain in the case of the iFrame on `line 35`. By having the `UseFor` enum we can select if we want to find the best domain for the iFrame or for the domain preview and add that early exit conditionally if `useFor === UseFor.IFrameDomain`